tasks/need_fixing_linters/golangci_lint_parser.py

```diff
@@ -0,0 +1,125 @@
+"""
+This module defines a parser for golangci-lint output.
+"""
+
+import re
+import os
+from codeowners import CodeOwners
+
+# example lint message
+# "pointer.go:6:1: package-comments: should have a package comment (revive)"
+LINT_PATTERN = re.compile("^([^:]+):([0-9]+):([0-9]+): (([^:]+): )?(.+) \\((.+)\\)$")
+
+# Example module message
+# "Linters for module /Users/pierre.gimalac/go/src/github.com/DataDog/datadog-agent/pkg/remoteconfig/state failed (base flavor)"
+MODULE_PATTERN = re.compile("^.*Linters for module ([^ ]+) failed \\((.+)\\).*$")
+
+CODEOWNERS_FILE_PATH = ".github/CODEOWNERS"
+
+def read_owners(owners_file: str):
+    """
+    Parses the CODEOWNERS file.
+    """
+    with open(owners_file, 'r', encoding="utf-8") as f:
+        codeowners = CodeOwners(f.read())
+    return codeowners
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏you could import the function from `tasks/libs/pipeline_notifications.py` rather than duplicating it

Reply:
Agreed ! Fixed in [db11ce2](https://github.com/DataDog/datadog-agent/pull/19015/commits/db11ce29d1a7577f4b2cae6b6bf144b7d7d6ae35).